apps/src/lib/node/ledger/mod.rs

```diff
@@ -353,16 +358,10 @@ async fn run_aux(config: config::Ledger, wasm_dir: PathBuf) {
                     abort_sender,
                 )
             }
-            #[cfg(feature = "eth-fullnode")]
-            ethereum_node::oracle::run_oracle(
-                ethereum_url,
-                eth_sender,
-                abort_sender,
-            )
```

Comment:
can this all be collpased into a single match case (more readable IMO), by changing the config struct to take an enum with the different options?

Reply:
sorry if it isn't clear what this is referring to, it's the entire `let oracle = ...` expr basically.

Reply:
yeah it would be better not to have mutually exclusive config options as well, will try this

Reply:
using an enum is much nicer!